packages/client/src/AccountsClient.js

```diff
@@ -177,8 +177,13 @@ export class AccountsClient {
   }
 
   resumeSession(): Promise<void> {
-    // TODO Should there be any additional resume session logic here?
-    return this.refreshSession();
+    const session = this.refreshSession();
+    session.then(() => {
+      if (this.options.onResumedSessionHook && isFunction(this.options.onResumedSessionHook)) {
+        this.options.onResumedSessionHook();
+      }
+    });
+    return session;
```

Comment:
If you're returning a value here then please update the return type of the this method to not be `Promise<void>`